Core/Core/UIViews/CoreSplitViewController.swift

```diff
@@ -131,14 +128,8 @@ extension HelmSplitViewController: UISplitViewControllerDelegate {
             var newDeets = nav.viewControllers[nav.viewControllers.count - 1]
```

Comment:
Maybe this could be renamed to something more expressing (some googling helped me figure out "deets" means "details", but still)

Reply:
I'll change it in the next PR.